oneanddone/tasks/models.py

```diff
@@ -34,6 +35,8 @@ def __unicode__(self):
 class Feedback(CachedModel, CreatedModifiedModel):
     attempt = models.OneToOneField('TaskAttempt')
     text = models.TextField()
+    how_many_minutes_did_you_spend_on_the_task = models.IntegerField(
```

Comment:
I think the old field name you used was better, except that it specified completed. I suggest changing this to `time_spent_in_minutes`. That is shorter and more succinct.